exchanges/btse/btse_wrapper.go

```diff
@@ -1051,11 +1051,16 @@ func (m *MarketPair) Pair() (currency.Pair, error) {
 	baseCurr := m.Base
 	var quoteCurr string
 	if m.Futures {
-		s := strings.Split(m.Symbol, m.Base) // e.g. RUNEPFC for RUNE-USD futures pair
-		if len(s) <= 1 {
-			return currency.EMPTYPAIR, errInvalidPairSymbol
+		if baseCurr == "TRUMPSOL" { // Only base currency which is different to the rest
+			baseCurr = "TRUMP"
+			quoteCurr = strings.TrimPrefix(m.Symbol, baseCurr)
+		} else {
+			s := strings.Split(m.Symbol, m.Base) // e.g. RUNEPFC for RUNE-USD futures pair
+			if len(s) <= 1 {
+				return currency.EMPTYPAIR, errInvalidPairSymbol
+			}
+			quoteCurr = s[1]
```

Comment:
Nit:
I think the same logic could apply for both if we just change baseCurr and what we split on.
Also: Are we worried about TRUMP(Some other cheap chain he decides is important and Btse has to jerry rig) ?

Solution for both, but can change to just use `== "TRUMPSOL"`:


```suggestion
		if strings.HasPrefix(baseCurr, "TRUMP") {
			baseCurr = "TRUMP"
		}
		s := strings.Split(m.Symbol, baseCurr) // e.g. RUNEPFC for RUNE-USD futures pair
```

Sorry; Suggested code won't be clean because it won't let me suggest over many lines..